### ISSUE TITLE
Create identification difficulty rule-set for eBMS verification

### ISSUE THREAD

**Issue by DavidRoy** [BiologicalRecordsCentre/ABLE#512]
extending #511 

Create a score for each species as to identification difficulty/evidence required to verify.  This can help with auto-verifying observations (i.e. for 'easy' to id species in known range and within flight periods).

UK scoring system is:
Grade 1: Species that can be relatively easily identified even by inexperienced recorders 
with the use of up-to-date field guides. Such records would be accepted without the need for 
additional evidence as long as they conform to the normal flight period and geographical 
range of the species.
Grade 2: Species where care is needed for identification. Should not present any difficulty for 
experienced recorders, but inexperienced recorders may be expected to provide additional 
evidence in the form of photographs of reasonable quality that show as many of the 
identification features as possible or, alternatively, a specimen. Even experienced recorders 
may be expected to provide additional evidence if the record is outside of the normal flight 
period or geographical range of the species. 
Grade 3: Species that are difficult to identify. Even experienced recorders may be expected 
to provide additional evidence, for example a specimen, particularly if the record is outside of 
the normal flight period or geographical range of the species.
Grade 4: Species that can only be identified following critical assessment, often involving 
dissection and examination of genitalia. A specimen should be retained for confirmation.

@chrisvanswaay do you think we could create something like for the European species and would it need to be regionalised?

**Comment by chrisvanswaay** [comment]
@DavidRoy Yes, I fear it will have to be regionalised. E.g. Melitaea athalia is easy in NL and UK, as it's the only species of its group, but in S Germany you can also get M parthenoides, M aurelia and M britomartis, and then it's a completely different story. So in the UK and NL this is easy, but not so in other parts of Europe. Not sure how to regionalise this efficiently, as it could be  a lot of work.

**Comment by DavidRoy** [comment]
ok, let's leave this for now as we have enough to do.  We'll concentrate on the geographic and phenology rules
